src/patternfly/components/Menu/examples/Menu.md

```diff
@@ -316,6 +316,37 @@ import './Menu.css'
 {{> menu-drilldown-example menu-drilldown-example--menu__content--attribute='style="--pf-c-menu--m-drilldown--Width: 350px;"'}}
```

Comment:
Should this one and the 3 examples above also change to `menu--Drilldown`?

Reply:
Nice catch! Updated the partial refs.